discovery-provider/src/models/delegates/delegation.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Boolean, Column, DateTime, ForeignKey, Integer, String, text
+from src.models.base import Base
+from src.models.model_utils import RepresentableMixin
+
+
+class Delegation(Base, RepresentableMixin):
+    __tablename__ = "delegations"
+
+    shared_address = Column(String, primary_key=True, nullable=False, index=True)
+    blockhash = Column(ForeignKey("blocks.blockhash"))  # type: ignore
+    blocknumber = Column(ForeignKey("blocks.number"))  # type: ignore
+    delegate_address = Column(String, nullable=False)
+    user_id = Column(Integer, nullable=False)
+    is_revoked = Column(Boolean, nullable=False, server_default=text("false"))
+    is_approved = Column(Boolean, nullable=False, server_default=text("false"))
+    is_current = Column(Boolean, nullable=False)
+    created_at = Column(DateTime, nullable=False)
+    updated_at = Column(DateTime, nullable=False)
+    txhash = Column(
+        String,
+        primary_key=True,
```

Comment:
don't think this is a primary key and server_default isn't necessary nullable is ok

Reply:
I set this, shared_address, and is_current as `primary_key=True` to denote that their combination is the primary key for the table. i think this is how it works in the other model files/tables as well, unless im misunderstanding?